### PR TITLE
Feature/icky 179

### PR DESCRIPTION
**Pull request by kibo-geetanshuchhabra**

### CODE REVIEW

File: framework/kibocommerce/api/endpoints/wishlist/get-wishlist.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  res,
+  body: { customerToken, includeProducts },
+  config,
+  commerce,
+}) => {
+  let result: { data?: any } = {}
+  if (customerToken) {
+    const customer = customerToken && (await getCustomerAccount({ customerToken, config }))
+    const customerId= customer?.id;
+    const wishlistName= customer?.id+ '-'+ customer?.firstName
```
Comment (kibo-kevinwatts) [inline framework/kibocommerce/api/endpoints/wishlist/get-wishlist.ts:16]: Just add a "defaultWishlistName" property to the config called 'My Wishlist'

Wishlist name does not need to be unique per shopper and using first name could cause issues if shopper changes account firstNmae

File: framework/kibocommerce/api/queries/get-customer-wishlist-query.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+       id
+        quantity
+        total
+        subtotal
+        unitPrice{
+          extendedAmount
+        }
+        quantity
+        product {
+          description
```
Comment (kibo-kevinwatts) [inline framework/kibocommerce/api/queries/get-customer-wishlist-query.ts:18]: Are there any fragments that could be re used here for Product? Maybe something shared with cart?

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]

File: framework/kibocommerce/api/endpoints/wishlist/get-wishlist.ts (hunk 1 above)
Comment (kibo-geetanshuchhabra) [reply framework/kibocommerce/api/endpoints/wishlist/get-wishlist.ts]: Resolved

File: framework/kibocommerce/api/queries/get-customer-wishlist-query.ts (hunk 2 above)
Comment (kibo-geetanshuchhabra) [reply framework/kibocommerce/api/queries/get-customer-wishlist-query.ts]: Resolved

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**